prog/dftb+/lib_dftbplus/main.F90

```diff
@@ -7519,8 +7519,8 @@ subroutine getReksEnProperties(eigenvecs, coord0, reks)
 
       call getUnrelaxedDensMatAndTdp(eigenvecs(:,:,1), reks%overSqr, reks%rhoSqrL, &
```

Comment:
Please pass reks instead of its components (the main code should know as little as possible about details of various extensions, as this more robust for future changes and extensions).

Reply:
@bhourahine I see, but I will fix this with other PR indicating refactoring for some REKS routines (I will create new PR after this PR is merged).